gcovr/tests/update-data/reference/coverage.update-data.h.html

```diff
@@ -0,0 +1,517 @@
+
+<html>
+
```

Comment:
Reference files like these seem to be a leftover from the time when we didn't separate references by compiler name. I think these should be removed.